client/src/utils/requestHandlers.js

```diff
@@ -0,0 +1,39 @@
+// Import Axios
+const axios = require("axios");
+
+// Import reducers
+// #NOTE I do not fully understand how to update the reducer. I know what is does, but not how to code its usage
```

Comment:
Reducers are scoped within the component that they're declared from, in this instance App.js. To add functionality, you inject `dispatch` to the code you need to call `dispatch` from.
That's why in `services/index.js` I have the partial application: 
```js
const userService = (body) => async (dispatch) => { ... }
```
And when it is called you would add:
```js
userService({...requestBody})(dispatch);
```